src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,62 @@
+.calendar {
+  flex-wrap: wrap;
+  display: flex;
+  gap: 1px;
+  padding: 10px;
+  width: 706px;
+}
+
+.calendar__day {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  width: $side-calendar-day;
+  height: $side-calendar-day;
```

Comment:
The variable `$side-calendar-day` is used here, but it's not defined within this file. Ensure that it is defined elsewhere in your SCSS files or imported correctly to avoid compilation errors.